Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/ScrollingViewIntegrationTest.java

```diff
@@ -281,24 +289,24 @@ public void dragAndCheckIfCanvasHasMovedInXAndY(PointF fromPoint, PointF toPoint
 	}
 
 	public void dragAndCheckIfCanvasHasMovedInXOrY(PointF fromPoint, PointF toPoint) {
-		int statusBarHeight = 0;
-		int resourceId = mainActivity.getResources().getIdentifier("status_bar_height", "dimen", "android");
-		if (resourceId > 0) {
-			statusBarHeight = mainActivity.getResources().getDimensionPixelSize(resourceId);
-		}
-
-		int actionBarHeight;
-		final TypedArray styledAttributes = mainActivity.getTheme().obtainStyledAttributes(
-				new int[] {android.R.attr.actionBarSize}
-		);
-		actionBarHeight = (int) styledAttributes.getDimension(0, 0);
-
-		PointF startPointSurface = new PointF(fromPoint.x, fromPoint.y + actionBarHeight + statusBarHeight);
+//		int statusBarHeight = 0;
+//		int resourceId = mainActivity.getResources().getIdentifier("status_bar_height", "dimen", "android");
+//		if (resourceId > 0) {
+//			statusBarHeight = mainActivity.getResources().getDimensionPixelSize(resourceId);
+//		}
+
+//		int actionBarHeight;
+//		final TypedArray styledAttributes = mainActivity.getTheme().obtainStyledAttributes(
+//				new int[] {android.R.attr.actionBarSize}
+//		);
+//		actionBarHeight = (int) styledAttributes.getDimension(0, 0);
```

Comment:
As you already ignored the tests, I would suggest to comment in all lines so the one who does the fixing of the tests knows which lines were in the test case initially.